linux/system.go

```diff
@@ -83,6 +84,25 @@ func (ls *linuxSystem) ScanDisks(filter disko.DiskFilter,
 	return disks, nil
 }
 
+func getDiskReadOnlyProperty(d disko.UdevInfo) (bool, error) {
+	syspath, err := getSysPathForBlockDevicePath(d.Name)
+	if err != nil {
+		return false, err
+	}
+
+	syspathReadOnly := syspath + "/ro"
+	if disko.PathExists(syspathReadOnly) {
+		content, err := ioutil.ReadFile(syspathReadOnly)
+		if err != nil {
+			return false, fmt.Errorf("failed to read %s for %s", syspathReadOnly, d.Name)
+		}
+		if string(content) == "1\n" {
```

Comment:
are there values other than 0 and 1? I think I'd just return error if there were other values.

Reply:
Just 1 or 0.